### PR TITLE
git-big-picture 1.2.2 (new formula)

### PR DESCRIPTION
**Pull request by hartwork**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Tests fail locally with mistaken(?) error `git-big-picture is missing test dependencies: graphviz sqlite python@3.11`. Let's see what the CI says.

Second take after #34826

CC @hanxue

### CODE REVIEW

File: Formula/git-big-picture.rb
Comment (carlocab) [inline Formula/git-big-picture.rb:3]:
```suggestion
  include Language::Python::Virtualenv

  desc "Visualization tool for Git repositories"
```

File: Formula/git-big-picture.rb
Comment (carlocab) [inline Formula/git-big-picture.rb:24]: `testpath` is already the working directory.
```suggestion
    system "git", "init"
    system "git", "config", "user.name", "Dummy author"
    system "git", "config", "user.email", "dummy@domain.test"
    system "git", "commit", "--allow-empty", "-m", "Empty commit"
    system "git", "big-picture", "-f", "svg", "-o", "output.svg"
    assert_path_exists testpath/"output.svg"
```
I'd try skipping the `git config` steps too, since I think `brew` sets that up automatically for the test.

Comment (hartwork) [reply Formula/git-big-picture.rb]: @carlocab done x2 :+1:

**Review by carlocab — APPROVED** [review]
Thanks!

**Comment by hartwork** [comment]
@carlocab thanks for the review! What does it take after approval to get this merged?

**Comment by carlocab** [comment]
We need CI to pass, but this was blocked on https://github.com/Homebrew/brew/pull/15301. That was just merged, so will re-run CI.

**Comment by hartwork** [comment]
> Ah, that. Please do this so we can get this merged.

@carlocab done. Are there any docs on why that is needed?

**Comment by carlocab** [comment]
> Are there any docs on why that is needed?

Not currently. This is relatively new to our merge flow.

> What is the bot trying to push to my branch?

Package metadata, basically.

**Comment by carlocab** [comment]
Thanks again for your contribution!

**Pull request merged**
